hack/update-infra-deployments.sh

```diff
@@ -33,7 +33,18 @@ RELEASE_POLICY_REF='quay.io/hacbs-contract/ec-release-policy:latest'
 function oci_source() {
   img="${1}"
   # Must use --raw because skopeo cannot handle an OPA bundle image format.
-  digest="$(skopeo inspect --raw  "docker://${img}" | sha256sum | awk '{print $1}')"
+  manifest="$(mktemp --tmpdir)"
+  function cleanup() {
+    # shellcheck disable=SC2317
+    rm "${manifest}"
+  }
+  trap cleanup RETURN
+  skopeo inspect --raw  "docker://${img}" > "${manifest}"
+  revision="$(jq -r '.annotations["org.opencontainers.image.revision"]' "${manifest}")"
+  if [[ -n "${revision}" && "${revision}" != null ]]; then
```

Comment:
It's a string null here right? For readability maybe:

```suggestion
  if [[ -n "${revision}" && "${revision}" != "null" ]]; then
```

Reply:
No need to quote non-space-containing strings in shell...

Reply:
Yeah, I know it's not needed, but when I read this code I spent a few moments wondering if a bare null had some significance in bash that I didn't remember. It doesn't of course, but we could consider optimizing to reduce brain cycles. :smile: